e2e/e2e_test.go

```diff
@@ -415,7 +426,51 @@ func TestE2E(t *testing.T) {
 				{
 					from: "sleep.naked", to: "httpbin.bar:8000", path: "/headers",
 					want:    []string{"HTTP/1.1 200 OK"},
-					notWant: []string{"By=spiffe://ymzhu-istio.svc.id.goog/ns/foo/sa/httpbin"},
+					notWant: []string{newResp("By=","foo", "httpbin")},
+				},
+			},
+		},
+		{
+			name: "mtls-port",
+			verify: []verifyCmd{
+				{
+					from: "sleep.foo", to: "httpbin.foo:9000", path: "/headers",
+					want: []string{newResp("By=","foo", "httpbin"), newResp("URI=","foo", "sleep")},
+				},
+				{
+					from: "sleep.foo", to: "httpbin.bar:9000", path: "/headers",
+					want: []string{newResp("By=","bar", "httpbin"), newResp("URI=","foo", "sleep")},
+				},
+				{
+					from: "sleep.bar", to: "httpbin.bar:9000", path: "/headers",
+					want: []string{newResp("By=","bar", "httpbin"), newResp("URI=","bar", "sleep")},
+				},
+				{
+					from: "sleep.bar", to: "httpbin.foo:9000", path: "/headers",
+					want: []string{newResp("By=","foo", "httpbin"), newResp("URI=","bar", "sleep")},
+				},
+				{
+					from: "sleep.naked", to: "httpbin.foo:9000", path: "/headers",
+					want: []string{"Connection reset by peer"},
+				},
+				{
+					from: "sleep.naked", to: "httpbin.bar:9000", path: "/headers",
+					want:    []string{"HTTP/1.1 200 OK"},
+					notWant: []string{newResp("By=","foo", "httpbin")},
+
+				},
+			},
+		},
+		{
+			name: "jwt-port",
```

Comment:
move this to the correct positions, the test case are ordered by the name, so this should be moved above.